mod1/mod11/src/file-3.xoo

```diff
@@ -0,0 +1,11 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// violate HasTag rule: xoo on line 11
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:36469/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:36469/project/issues?id=sample&pullRequest=6367&issues=AWkP5Iqw29c0llRUez2-&open=AWkP5Iqw29c0llRUez2-)

[ISSUE_KEY]: AWkP5Iqw29c0llRUez2-